platform/src/test/java/org/hillview/test/storage/SSTableTest.java

```diff
@@ -114,4 +129,41 @@ public void testLazyLoading() {
             this.ignoringException("Failed to read SSTable (" + this.ssTablePath + ")", e);
         }
     }
+
+    // @Test
+    /** Shows the interaction between CassandraDatabase.java and CassandraSSTableLoader.java */
+    public void TestCassandraDatabase() {
+        try {
+            // Connecting to Cassandra node and get some data
+                CassandraDatabase cassDB = new CassandraDatabase(this.info);
+                cassDB.loadKeyspacesAndTables();
+                cassDB.printCassTables();
+                cassDB.loadTablePartition("cassdb");
+                cassDB.printTablePartition();
+                long count = cassDB.getRowCount("cassDB", "flights");
+                System.out.println("Count: " + count + " rows");
+
+                String ssTablePath = cassDB.getSSTablePath("cassdb", "flights");
```

Comment:
Since we put the database name and table name in the ConnectionInfo, I believe the client will provide us the correct information regarding those two fields. Do we still need a method to discover the available databases and tables in the cluster? Perhaps we need to display this databases to the client so that they can choose. 

The table partition will be used by SSTableReader to load the local key-range only. These three methods are independent of each other: loadKeyspacesAndTables(), loadTablePartition(), and getSSTablePath()

Reply:
I thought about this, and having a browser could be useful, but for now it's not our most urgent feature, so let's assume that the users can use other Cassandra tools to browse the database. So let's keep the implementation as simple as possible.